Auth0/Response.swift

```diff
@@ -57,7 +57,7 @@ struct Response<E: Auth0Error> {
             return json
         }
         // This piece of code is dedicated to our friends the backend devs :)
-        if response.url?.lastPathComponent == "change_password" {
+        if response.url?.lastPathComponent == "change_password" || response.url?.lastPathComponent == "revoke" {
```

Comment:
It's deal with in Guardian, not so transferrable https://github.com/auth0/Guardian.swift/blob/master/Guardian/Request.swift#L88

I looked at the `changed_password` response, forgot it returned plain text.  
I've made a change, feel free to tell me if it sucks :).